dll/win32/iernonce/include/registry.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     GPL - See COPYING in the top level directory
+ * FILE:        dll\win32\iernonce\registry.cpp
+ * PURPOSE:     Functions to read RunOnceEx registry.
+ * PROGRAMMERS: Copyright 2013-2016 Robert Naumann
+ */
+
+
+#include <windows.h>
+#include <set>
+#include <map>
+#include <atlbase.h>
+#include <atlstr.h>
+#include <atlcoll.h>
+
+
+typedef class RunOnceExSection *pRunOnceExSection;
+
+
+class CRegKeyEx : public CRegKey
+{
+public:
+    LONG EnumValueName(
+        DWORD iIndex,
+        LPTSTR pszName,
+        LPDWORD pnNameLength);
+};
+
+struct RunOnceExEntryCmp {
+    bool operator()(const ATL::CStringW &lhs, const ATL::CStringW &rhs) const;
+};
+
+struct RunOnceExSectionCmp {
+    bool operator()(const pRunOnceExSection &lhs, const pRunOnceExSection &rhs) const;
+};
+
+typedef std::map<ATL::CStringW, ATL::CStringW, RunOnceExEntryCmp> RNONCE_ENRTY_MAP;
+typedef std::set<RunOnceExSection *, RunOnceExSectionCmp> RNONCE_SECTION_SET;
+
+class RunOnceExSection
+{
+private:
+    ATL::CStringW m_SectionName;
+    RNONCE_ENRTY_MAP m_EntryMap;
```

Comment:
`RUNONCE_ENTRY_MAP`

Reply:
I'm trying to shorten the namings, and the library itself is called `iernonce` so I think this naming is ok 🤣
however if you prefer `RUNONCE_ENTRY_MAP` I can rename it

Reply:
Yes the filename uses a 8.3 format; but it's not a reason to make names unreadable within the code 😝

Reply:
The name iernonce seems to be a relic from 8dot3 times. IMHO RUNONCE_ENTRY_MAP is more readable.

Reply:
sounds reasonable! I will rename it.